Vagrantfile

```diff
@@ -151,8 +205,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         },
         "masakari_controller_conf" => {
           "hostentrylist" => [
-            { "compute1" => "192.168.50.11" },
-            { "compute2" => "192.168.50.12" }
+            { "compute01" => "192.168.50.11" },
```

Comment:
If you change the hostname, you should adjust the description in README.md as well.